samples/excel/50-workbook/create-workbook.yaml

```diff
@@ -0,0 +1,94 @@
+order: 2
+id: excel-workbook-create-workbook
+name: Create workbook
+description: Creates new workbooks both from scratch and by copying an existing one.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.8
+script:
+    content: |
+        $("#create-new-blank-workbook").click(() => tryCatch(createBlankWorkbook));
+        $("#file").change(() => tryCatch(createWorkbookFromExisting));
+
+        async function createBlankWorkbook() {
+            await Excel.run(async (context) => {
+                Excel.createWorkbook();
+            });
+        }
+
+        async function createWorkbookFromExisting() {
+            const myFile = document.getElementById("file");
+            const reader = new FileReader();
+
+            reader.onload = ((event) => {
+                Excel.run(context => {
+                    // strip off the metadata before the base64-encoded string
+                    const startIndex = event.target.result.indexOf("base64,");
+                    const mybase64 = event.target.result.substr(startIndex + 7);
+
+                    Excel.createWorkbook(mybase64);
+                    return context.sync();
+                });
+            });
+
+            // read in the file as a data URL so we can parse the base64-encoded string
+            reader.readAsDataURL(myFile.files[0]);
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to create a new workbook by copying an existing one.</p>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="create-new-blank-workbook" class="ms-Button">
+                <span class="ms-Button-label">Create new blank workbook</span>
+            </button><p>
```

Comment:
Suggest that you add a heading and a sentence above the button (to make it clearer visually the the snippet lets you do two different things -- 1) create empty workbook, 2) copy existing workbook) and change the button label to `Create workbook`

```
<p><b>Create empty workbook</b></p>
<p>Press the button to create a new (empty) workbook and open it in a new instance of Excel.</p>
<button id="create-new-blank-workbook" class="ms-Button">
    <span class="ms-Button-label">Create workbook</span>
</button><p>

Reply:
Added a new header. I like the pattern of a header per distinct scenario. I found adding more text to be redundant. The button is self-explanatory, especially with the snippet description.